src/tagmanager/tm_workspace.c

```diff
@@ -695,7 +695,7 @@ static void fill_find_tags_array_prefix(GPtrArray *dst, const GPtrArray *src,
 	for (i = 0; i < count && num < max_num; ++i)
 	{
 		if (tm_parser_langs_compatible(lang, (*tag)->lang) &&
-			!tm_tag_is_anon(*tag) &&
+			!((*tag)->flags & tm_tag_flag_anon_t) &&
```

Comment:
IMO the this is not an improvement. Might keep `tm_tag_is_anon()` and reduce its implementation to just test the flag

Reply:
Yeah, you are right, looking at it now side by side, this is much less readable. Will fix.